spec/services/inherited_proofing/va/service_spec.rb

```diff
@@ -27,21 +27,25 @@
       let(:auth_code) { 'mocked-auth-code-for-testing' }
 
       it 'makes an authenticated request' do
-        stub = stub_request(:get, request_uri).
-          with(headers: request_headers).
-          to_return(status: 200, body: '{}', headers: {})
+        freeze_time do
+          stub = stub_request(:get, request_uri).
+            with(headers: request_headers).
+            to_return(status: 200, body: '{}', headers: {})
 
-        service.execute
+          service.execute
```

Comment:
That space supposed to be there? I can’t tell I’m doing this on my phone.

Reply:
Yep, I think it's from the indent within the `freeze_time` block